dayinfo.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+from calendar import isleap
+
+from pynameday.slovakia import Slovakia as sknameday
+from workalendar.europe import Slovakia as skholiday
+
+
+INTERNATIONAL_DAY = {
+    '1': {'27': 'International Day of Commemoration in Memory of the Victims \
+                 of the Holocaust'},
+
+    '2': {'4': 'World Cancer Day',
+          '8': 'Day of Safer Internet',
+          '11': 'World Day of the Sick',
+          '14': 'World Valentine Day',
+          '15': 'World Single’s Day',
+          '21': 'International Mother Language Day'},
+
+    '3': {'8': 'International Women’s Day',
+          '14': 'Pi Day',
+          '20': 'International Day of Happiness',
+          '28': 'Day of Teachers'},
+
+    '4': {'11': 'Daffodil Day - the day of the fight against cancer',
+          '22': 'International Mother Earth Day'},
+
+    '5': {'15': 'International Family Day',
+          '17': 'World Telecommunication and Information Society Day'},
+
+    '6': {'1': 'International Children’s Day',
+          '21': 'Day of flowers'},
+
+    '7': {'8': 'Day of Videogames',
+          '30': 'International Day of Friendship'},
+
+    '8': {'13': 'International Lefthanders Day'},
+
+    '9': {'13': 'International Day of Programmers',
+          '19': 'Day of Software Freedom'},
+
+    '10': {'5': 'World Teachers’ Day',
+           '20': 'International Day of Trees'},
+
+    '11': {'2': 'All Souls Day',
+           '17': 'International Students Day',
+           '30': 'Day of Computer Safety'},
+
+    '12': {'10': 'Human Rights Day'}
+}
+
+
+if isleap(datetime.today().year):
+    del INTERNATIONAL_DAY['8']['13']
+    INTERNATIONAL_DAY['8']['12'] = 'International Day of Programmers'
+
+
+def formatter(string, prefix=''):
+    """Format data"""
+    if string:
+        return ', {0}{1}'.format(prefix, string)
+    return ''
+
+
+@cmd
+def dayinfo(event):
+    """Inform you about current day"""
```

Comment:
@Adman can you please update this docstring with an example such as here: https://github.com/mrshu/brutal-plugins/blob/master/weather.py#L92

Also, did you perhaps want to say "Informs you about current day"?